proxy/build.gradle

```diff
@@ -68,6 +68,8 @@ dependencies {
 
     implementation(platform("net.kyori:adventure-bom:${adventureVersion}"))
     implementation("net.kyori:adventure-nbt")
+    implementation(platform("net.kyori:adventure-bom:${adventureVersion}"))
```

Comment:
You have missed a duplicate dependency of adventure-bom here

Reply:
Removed